src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/BulkPrintController.java

```diff
@@ -55,41 +58,61 @@
             @ApiResponse(code = 500, message = "Internal Server Error")
         })
     public ResponseEntity<AboutToStartOrSubmitCallbackResponse> bulkPrint(
-        @RequestHeader(value = AUTHORIZATION_HEADER, required = false) String authorisationToken,
+        @RequestHeader(value = AUTHORIZATION_HEADER, required = false) String authToken,
         @NotNull @RequestBody @ApiParam("Callback") CallbackRequest callback) {
 
         long caseId = callback.getCaseDetails().getId();
         log.info("Received request for bulk print for Case ID : {}", caseId);
 
         validateCaseData(callback);
 
-        Map<String, Object> caseData = callback.getCaseDetails().getData();
-        CaseDocument coverSheetRes = coverSheetService.generateRespondentCoverSheet(callback.getCaseDetails(), authorisationToken);
-        UUID letterIdRes = bulkPrintService.sendForBulkPrint(coverSheetRes, callback.getCaseDetails());
+        caseDetails = callback.getCaseDetails();
```

Comment:
why don't we pass `caseDetails` to all methods. I don't like this approach of not-global-but-global private var.

Reply:
I think what @yarpo means is let's use this as local variable (local to the method).
I agree.